traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -174,7 +175,21 @@ func Read(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		resolvers = append(resolvers, resolver)
+		// Based on version we load types - for version 5 and above we use FederationResolverV5
+		if inf.Version.GreaterThanOrEqualTo(&api.Version{Major: 5, Minor: 0}) {
+			var v5Resolver tc.FederationResolverV5
+
+			// Convert FederationResolver fields to FederationResolverV5 fields
+			v5Resolver.ID = resolver.ID
+			v5Resolver.IPAddress = resolver.IPAddress
+			lastUpdated := time.Unix(resolver.LastUpdated.Unix(), 0)
+			v5Resolver.LastUpdated = &lastUpdated
+			v5Resolver.Type = resolver.Type
+
+			resolvers = append(resolvers, &v5Resolver)
```

Comment:
Should this be appending the reference of v5Resolver?

Reply:
here we are trying to append value of v5Resolver to resolvers, is that done wrongly ?

Reply:
It needs to append the value of v5Resolver, not the reference (address), which is what the `&` does.
So, it should just be 
```resolvers = append(resolvers, v5Resolver)